.env

```diff
@@ -0,0 +1,5 @@
+POSTGRES_USER=''
+POSTGRES_PASSWORD=''
+POSTGRES_HOST='localhost'
+POSTGRES_DB='final_project_db'
+POSTGRES_TEST_DB='final_project_test'
```

Comment:
[Optional]

I would recommend not to check these type of files to git.

See [this article](https://dev.to/somedood/please-dont-commit-env-3o9h) and [this discussion](https://github.com/bkeepers/dotenv/issues/67) for more details.

Reply:
Fixed, .env file added to .gitignore